test/e2e/node/security_context.go

```diff
@@ -146,37 +147,131 @@ var _ = SIGDescribe("Security Context", func() {
 		pod := scTestPod(false, false)
-		pod.Annotations[v1.SeccompContainerAnnotationKeyPrefix+"test-container"] = "unconfined"
+		pod.Annotations[v1.SeccompContainerAnnotationKeyPrefix+"test-container"] = v1.SeccompProfileNameUnconfined
 		pod.Annotations[v1.SeccompPodAnnotationKey] = v1.SeccompProfileRuntimeDefault
 		pod.Spec.Containers[0].Command = []string{"grep", "ecc", "/proc/self/status"}
 		f.TestContainerOutput(v1.SeccompPodAnnotationKey, pod, 0, []string{"0"}) // seccomp disabled
 	})
 
-	ginkgo.It("should support seccomp alpha unconfined annotation on the pod [Feature:Seccomp] [LinuxOnly]", func() {
-		// TODO: port to SecurityContext as soon as seccomp is out of alpha
+	ginkgo.It("should support seccomp unconfined on the pod by annotation [LinuxOnly]", func() {
 		pod := scTestPod(false, false)
-		pod.Annotations[v1.SeccompPodAnnotationKey] = "unconfined"
+		pod.Annotations[v1.SeccompPodAnnotationKey] = v1.SeccompProfileNameUnconfined
 		pod.Spec.Containers[0].Command = []string{"grep", "ecc", "/proc/self/status"}
 		f.TestContainerOutput(v1.SeccompPodAnnotationKey, pod, 0, []string{"0"}) // seccomp disabled
 	})
 
-	ginkgo.It("should support seccomp alpha runtime/default annotation [Feature:Seccomp] [LinuxOnly]", func() {
-		// TODO: port to SecurityContext as soon as seccomp is out of alpha
+	ginkgo.It("should support seccomp runtime/default by annotation [LinuxOnly]", func() {
 		pod := scTestPod(false, false)
 		pod.Annotations[v1.SeccompContainerAnnotationKeyPrefix+"test-container"] = v1.SeccompProfileRuntimeDefault
 		pod.Spec.Containers[0].Command = []string{"grep", "ecc", "/proc/self/status"}
 		f.TestContainerOutput(v1.SeccompPodAnnotationKey, pod, 0, []string{"2"}) // seccomp filtered
 	})
 
-	ginkgo.It("should support seccomp default which is unconfined [Feature:Seccomp] [LinuxOnly]", func() {
-		// TODO: port to SecurityContext as soon as seccomp is out of alpha
+	ginkgo.It("should support seccomp unconfined on the container by field [LinuxOnly]", func() {
 		pod := scTestPod(false, false)
+		pod.Spec.SecurityContext.SeccompProfile = &v1.SeccompProfile{Type: v1.SeccompProfileTypeRuntimeDefault}
+		pod.Spec.Containers[0].SecurityContext.SeccompProfile = &v1.SeccompProfile{Type: v1.SeccompProfileTypeUnconfined}
 		pod.Spec.Containers[0].Command = []string{"grep", "ecc", "/proc/self/status"}
 		f.TestContainerOutput(v1.SeccompPodAnnotationKey, pod, 0, []string{"0"}) // seccomp disabled
 	})
+
+	ginkgo.It("should support seccomp unconfined on the pod by field [LinuxOnly]", func() {
+		pod := scTestPod(false, false)
+		pod.Spec.SecurityContext.SeccompProfile = &v1.SeccompProfile{Type: v1.SeccompProfileTypeUnconfined}
+		pod.Spec.Containers[0].Command = []string{"grep", "ecc", "/proc/self/status"}
+		f.TestContainerOutput(v1.SeccompPodAnnotationKey, pod, 0, []string{"0"}) // seccomp disabled
+	})
+
+	ginkgo.It("should support seccomp runtime/default on the container by field [LinuxOnly]", func() {
```

Comment:
nit: maybe this better translates the intention behind this test
```suggestion
	ginkgo.It("should overwrite pod-level profile with 'runtime/default' from container seccomp profile field [LinuxOnly]", func() {
```

Reply:
Fixed as suggested :+1: